### PR TITLE
add warning if evaluation on only one estimate

### PR DESCRIPTION
**Pull request by Chadys**
Fix issue #52
I added a warning plus corresponding test, I also corrected `prepare_tests.sh` because paths were incorrect : 'data/' is not needed since a `cd data` is done before. Tell me if I should create two separate pull request instead for this issue.

### CODE REVIEW

File: museval/__init__.py [hunk 1]
```diff
@@ -365,6 +365,14 @@ def eval_mus_track(
                 target_name=target,
                 values=values
             )
+    elif not has_acc:
+        import warnings
+
+        warnings.warn(
+            UserWarning(
+                "Incorrect usage of BASS : at least two estimates must be provided. Target score will be empty."
```
Comment (faroit) [inline museval/__init__.py]: Maybe `BSSeval` instead of `BASS`? (what does `BASS stand for?)

File: museval/__init__.py [hunk 2]
```diff
@@ -365,6 +365,14 @@ def eval_mus_track(
                 target_name=target,
                 values=values
             )
+    elif not has_acc:
+        import warnings
```
Comment (faroit) [inline museval/__init__.py]: import at the top

File: tests/test_regression.py [hunk 3]
```diff
@@ -79,3 +79,25 @@ def test_estimate_and_evaluate(mus):
             )
 
             assert np.allclose(ref, est)
+
+
+def test_one_estimate(mus):
+    track = mus.load_mus_tracks(
+        tracknames=[os.path.splitext(os.path.basename(json_path))[0]]
```
Comment (faroit) [inline tests/test_regression.py]: would need to changed for #56

**Comment by faroit** [comment]
This is great. Thanks. 

I am currently working on a bigger update. 
Either I will merge this when #56 is ready (which would require changes). or you can directly change to the target branch to `musdb3_compability`

**Comment by Chadys** [comment]
I changed the target branch and made the needed changes

**Comment by faroit** [comment]
Great. Thanks a lot!

**Pull request merged**